engine/src/main/java/org/terasology/rendering/dag/nodes/CopyImageToScreenNode.java

```diff
@@ -60,12 +65,15 @@ public void initialise() {
     public void process() {
         PerformanceMonitor.startActivity("rendering/copyImageToScreen");
         sceneFinalFbo.bindTexture(); // TODO: Convert to a StateChange
```

Comment:
The slot is likely to be 0 but you should check the relevant ShaderParameters class.
The materialUrn is the ResourceUrn uniquely identifying the material/shader being enabled via EnableMaterial. The materialParameter is the name of the sampler variable in the shader receiving the texture.

I'm ok if you'd like to change these type of lines altogether rather than here.